src/lfc_scripts.py

```diff
@@ -1,13 +1,61 @@
 import mwclient
 import config
+from geopy.geocoders import Nominatim
+import folium
 
-site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
-site.login(config.username, config.api_key)
+def main():
+    site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
+    site.login(config.username, config.api_key)
 
-response = site.api(
-    'torquedataconnect',
-    format='json',
-    path='/competitions/LoneStar2020/proposals'
-)
+    response = site.api(
+        'torquedataconnect',
+        format='json',
+        path='/competitions/LoneStar2020/proposals'
+    )
+    proposal_ids = response["result"]
 
-print(response["result"]) 
\ No newline at end of file
+    geolocator = Nominatim(user_agent = "map")
+
+    #Extract organization locations
+    locations = []
+    for id in proposal_ids:
+
+        response = site.api(
+            'torquedataconnect',
+            format='json',
+            path='/competitions/LoneStar2020/proposals' + '/' + id
+        )
+        response['result']
+        org_location = geolocator.geocode(concat_org_location(response['result'])) #Convert named address to latlong pair
+        try:
+            coordinate_pair = (org_location.latitude, org_location.longitude)
+            print(org_location.latitude, org_location.longitude)
+        except:
+            print('error, not a location')
+        locations.append(coordinate_pair)
+
+    #Plot latlong points
```

Comment:
You should have a space after your `#` in comments

so `# Plot` instead of `#Plot`

You can see the Python comment style guide here: https://www.python.org/dev/peps/pep-0008/#comments

^ this applies to other inline comments and should be fixed there too.

Reply:
Completed in edits.